spec/views/catalog/_show_default.html.erb_spec.rb

```diff
@@ -24,4 +27,19 @@
     expect(rendered).to have_content("Alternative")
     expect(rendered).to have_content("Test")
   end
+
+  describe "catalog/_small_image_viewer" do
```

Comment:
Im curious if this should be in its own view spec (i.e spec/views/_small_image_viewer.html.erb_spec.rb)
@terrellt ?